### DISCUSSION TITLE
Multiple layouts for collection

### DISCUSSION THREAD

**Discussion by ithan** [directus/directus#19915] [Feature Requests]
### Summary

Allow to have custom layouts ( and with that I mean different selected fields that are displayed ) on each collection, and allow the user to select what layout they want to display.

### Basic Example

If I have 1 collection, lets call it feedback

in feedback I may have 20 columns, 11 relevant for 1 team, and 9 relevant for another team, but all part of the feedback collection itself.

Each team wants to display only some information, but currently you need either and insight for each team or getting the data into another tool to display it

### Motivation

Enhance user experience by allowing custom layouts for collections, empowering teams to view data that's most pertinent to them without external tool dependencies.

### Detailed Design

The design introduces a "Custom Layout" option for each collection, enabling users to select which fields/columns they wish to display. The layouts are saved as user or team preferences, allowing quick switching between different views.
Components:

    Custom Layout Interface: A user-friendly interface within the collection view where users can select, arrange, and save the desired columns.

    Layout Storage: Store each layout configuration either as a user-specific setting or a team-specific setting.

    Layout Switcher: A dropdown or similar UI element to switch between different saved layouts quickly.

### Must Have:
- A list of layouts for the user to select
- A way for the user to select the layout

### Could Have:
- Permission for who can create new layouts
- Stats to find what layout is most used 

### Drawbacks

This may require to store quite a bit of information for each layout, and it will need to take into account situations where the columns change 

### Alternatives

- I'm not sure about this one :D 

### Adoption Strategy

Since this is mostly a UI change, it should not instruct any breaking change

**Comment by rijkvanzanten** [comment]
Heya! This is where Bookmarks come into play! You can setup different layout configurations and save it as a bookmark to allow users to switch back and forth between the various preconfigured layout options. Bookmarks can be configured globally, per role, or per user.

I'll close this for now as I believe this functionality is already available, but happy to reopen / keep discussing if I misunderstood 🙂
